src/layouts/shortcodes/gsctl_version.html

```diff
@@ -1 +1 @@
-1.1.6
\ No newline at end of file
+null
```

Comment:
@giantswarm/sig-docs Having `null`/`ull` here seems to be a bug

Reply:
Yes, we have this issue for it https://github.com/giantswarm/roadmap/issues/2548

Locally everything works for me, but in CI it fails. Couldn't find any reason.